packages/grid/x-data-grid/src/constants/localeTextConstants.ts

```diff
@@ -3,6 +3,7 @@ import { GridLocaleText } from '../models/api/gridLocaleTextApi';
 export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   // Root
   noRowsLabel: 'No rows',
+  noUnpinnedRowsOverlayLabel: 'No unpinned rows found.',
   noResultsOverlayLabel: 'No results found.',
```

Comment:
Nitpick but the dot at the end of those labels seems a bit off to me. In general our labels don't take final dots because they're not really sentences, I'm wondering why we have one here.

Reply:
Good question, Idk to be honest but it looks more correct punctuation wise 🤔